src/models/index.ts

```diff
@@ -17,6 +17,7 @@ export { VerticalSearchState, AllResultsForVertical } from './slices/vertical';
 export { BoundedRange, RangeBoundary } from './utils/boundedrange';
 export { SelectableFilter } from './utils/selectablefilter';
 export { SearchTypeEnum, SearchType } from './utils/searchType';
+export { EnumOrLiteral } from '../utils/types';
```

Comment:
Should this export be in src/index.ts instead? I thought src/models/index.ts should only export things that are within the `models` folder

Reply:
good point, I moved it